### ISSUE TITLE
Ask add-on source code when responding yes to obfuscated code

### ISSUE THREAD

**Issue by lafonth** [mozilla/addons-server#13416]
When a dev is submitting an add on and is responding yes to the question of the obfuscated code, the form is proposing to submit source code. 
Obfuscated code is totally forbidden by our Add-on policy today.

saw with @kewisch

**Comment by eviljeff** [comment]
@jvillalobos / @wagnerand what should the copy be revised to?

**Comment by wagnerand** [comment]
Turns out, there are several strings that would need to be updated:

```
src/olympia/devhub/templates/devhub/addons/submit/done.html
53:           "if your submission includes minified, obfuscated or compiled code.") }}

src/olympia/devhub/templates/devhub/includes/source_form_field.html
3:    {{ _('If your add-on contains binary or obfuscated code other than known '

src/olympia/devhub/templates/devhub/versions/edit.html
126:              {{ tip(None, _("If your add-on contain binary or obfuscated code, make the source available here for reviewers.")) }}

static/js/common/upload-addon.js
474:                            messages.push(gettext("Compiled binaries, as well as minified or obfuscated scripts (excluding known libraries) need to have their sources submitted separately for review. Make sure that you use the source code upload field to avoid having your submission rejected."));
```

I think we should generally refer to it as "minified, concatenated or otherwise machine-generated code". @kewisch what do you think?

**Comment by jvillalobos** [comment]
The proposed change look good to me.

**Comment by ManishAradwad** [comment]
Would like to work on this one...
I'm supposed to change only the prompt being displayed, right??

**Comment by varunbankar** [comment]
Would like to work on this
So as far I understand the word "obfuscated" needs to be changed to "minified, concatenated or otherwise machine-generated code"?

**Comment by caitmuenster** [comment]
Mentor: @bobsilverberg 

@ManishAradwad, it looks like you commented on this issue first. Do you still want to work on this issue?

**Comment by ManishAradwad** [comment]
Sure, I'd like to give it a try...

**Comment by AlexandraMoga** [comment]
I've checked the pages where the new terminology was updated and I could find two of them:

1. In the last submission step:
![image](https://user-images.githubusercontent.com/31961530/76202771-0dd6f480-61fe-11ea-879f-0f0d1e74b598.png)

2. On the manage version page (Source code field tooltip)
![image](https://user-images.githubusercontent.com/31961530/76202794-18918980-61fe-11ea-92d0-0d080335fd30.png)

As for the third string updated, i.e. `src/olympia/devhub/templates/devhub/includes/source_form_field.html`, this belongs to the old source code submission method. Source code submission has its own dedicated page now and the change should be made to `/src/olympia/devhub/templates/devhub/addons/submit/source.html`
I've filed #13677 as a follow up.
